ui/tests/integration/components/oidc/key-form-test.js

```diff
@@ -49,7 +49,6 @@ module('Integration | Component | oidc/key-form', function (hooks) {
     />
     `);
 
-    assert.dom('[data-test-oidc-key-title]').hasText('Create Key', 'Form title renders correct text');
```

Comment:
I deleted these assertions instead of move them because they no longer rely on `@model.isNew` logic and are hardcoded